src/openforms/translations/api/views.py

```diff
@@ -1,23 +1,34 @@
+from pathlib import Path
+from typing import List
+
 from django.conf import settings
+from django.http import FileResponse
 from django.utils.translation import (
     activate,
     get_language,
     get_language_info,
     gettext_lazy as _,
 )
 
-from drf_spectacular.utils import OpenApiExample, extend_schema
-from rest_framework import status
+from drf_spectacular.plumbing import build_object_type
+from drf_spectacular.utils import OpenApiExample, OpenApiParameter, extend_schema
+from rest_framework import permissions, status
+from rest_framework.exceptions import NotFound
 from rest_framework.request import Request
 from rest_framework.response import Response
 from rest_framework.views import APIView
 
+from openforms.api.drf_spectacular.functional import lazy_enum
 from openforms.api.serializers import ExceptionSerializer, ValidationErrorSerializer
 from openforms.translations.utils import set_language_cookie
 
 from .serializers import LanguageCodeSerializer, LanguageInfoSerializer
 
 
+def get_language_codes() -> List[str]:
+    return [language[0] for language in settings.LANGUAGES]
+
```

Comment:
Maybe reuse this in the other view that references this same list `LanguageInfoView.get()`.

Reply:
done that!